Makefile

```
@@ -0,0 +1,53 @@
+GO_VERSION ?= 1.20.2
+GOLANG_CI_VER ?= v1.52
+GOSEC_VER ?= 2.15.0
+TEST_COVERAGE_FILE=lcov.info
+TEST_COVERAGE_HTML_FILE=coverage_unit.html
+TEST_COVERAGE_FUNC_FILE=func_coverage.out
+
+# CONTAINER_RUNNABLE checks if tests and lint check can be run inside container.
+PODMAN ?= $(shell podman -v > /dev/null 2>&1; echo $$?)
+ifeq ($(PODMAN), 0)
+CONTAINER_RUNTIME=podman
+else
+CONTAINER_RUNTIME=docker
+endif
```

@henderiw @vjayaramrh what about something like this?
```suggestion
CONTAINER_RUNTIME ?= $(shell which docker 2> /dev/null || which podman 2> /dev/null || echo docker)
```

---

@electrocucaracha I believe that podman is installed but the podman machine is stopped, hence the issue is being observed.

---

I reinstalled Podman and now it is ok, so this was a local machine issue açais.

---

/lgtm